AWS/legos/aws_delete_ebs_snapshot/aws_delete_ebs_snapshot.py

```diff
@@ -0,0 +1,42 @@
+##
+##  Copyright (c) 2023 unSkript, Inc
+##  All rights reserved.
+##
+from pydantic import BaseModel, Field
+from typing import List, Dict
+import pprint
+
+class InputSchema(BaseModel):
+    region: str = Field(
+        title='Region',
+        description='AWS Region.')
+
+    snapshot_id: str = Field(
+        title='Snapshot ID',
+        description='EBS snapshot ID. Eg: "snap-34bt4bfjed9d"')
+
+
+def aws_delete_ebs_snapshots_printer(output):
+    if output is None:
+        return
+    pprint.pprint(output)
+
+
+def aws_delete_ebs_snapshots(handle, region: str, snapshot_id: str) -> Dict:
```

Comment:
can we be careful with the name, this is only deleting one snapshot, right, so please match the name of the function to the directory and filename

Reply:
sure...